src/Common/Constants.ts

```diff
@@ -362,6 +364,13 @@ export class Notebook {
     "We have identified an issue with the Mongo Shell and it is unavailable right now. We are actively working on the mitigation.";
   public static readonly cassandraShellTemporarilyDownMsg =
     "We have identified an issue with the Cassandra Shell and it is unavailable right now. We are actively working on the mitigation.";
+  public static saveNotebookModalTitle = "Save Notebook in temporary environment";
+  public static saveNotebookModalContent =
```

Comment:
>saveNotebookModalContent [](http://example.com/codeflow?start=16&length=24)

not blocking this checkin: we ll have to revise this string if needed. dedicated period doesnt tell me as a customer - how much time.

Reply:
ok